datavirt64/dynamicvdb-datafederation/derby/README.md

```diff
@@ -1,15 +1,26 @@
-##Overview
+## Overview
 
-This directory contains details specific to using this quickstart with a Derby database.  It contains the [customer-schema.sql](./customer-schema.sql) file which includes the DDL and some seed data, along with this readme.
+This directory contains details specific to using this quickstart with a Derby database.  It contains the [customer-schema.sql](./customer-schema.sql) file which includes the DDL and some seed data, along with this readme.  The customer-schema.sql is provided for informational purposes or for your use if you would like to setup your own database instance that contains the same data.
 
-##Initializing the Database
+## Initializing the Database
 
 No extra steps are required to initialize the database as it is injected into the image during the s2i build (from [../app/data/databases/derby](../app/data/databases/derby)).
 
-##Running the QuickStart
+## Running the QuickStart
+
+1. Data Source Configuration
 
 The Derby quickstart requires the Derby JDBC driver to be installed in the JDV image.  This is accomplished by creating a image that contains the driver, as well as an installation script, which adds the driver configuration to the JDV configuration.  An example of a Derby driver image can be found [here](../../derby-driver-image).
 
+
+2. Create the app-secret
+
+`datavirt-app-config` secret is defined and linked to the service account when that file is installed (e.g. `oc create -f application-templates/secrets/datavirt-app-secret.yaml`).
+    *  `oc create -f application-templates/secrets/datavirt-app-secret.yaml`
```

Comment:
shouldn't this reference the jdv image repo now?

Reply:
yes